test/dynamo/test_functions.py

```diff
@@ -1136,11 +1143,11 @@ def test_device(x):
         if not x.is_cuda:
             return x + 1
 
-    @unittest.skipIf(not torch.cuda.is_available(), "requires cuda")
+    @unittest.skipIf(not HAS_GPU, "requires gpu")
     @make_test
     def test_get_device_properties_tensor_device(a):
-        x = a.to("cuda")
-        prop = torch.cuda.get_device_properties(x.device)
+        x = a.to(device_type)
+        prop = torch.get_device_module(device_type).get_device_properties(x.device)
         if prop.major == 8:
```

Comment:
`multi_processor_count` is a cuda specific device property. Let's leave this UT cuda-specific.